lib/ansible/modules/cloud/oracle/oci_vcn.py

```diff
@@ -23,7 +20,7 @@
     - This module allows the user to create, delete and update virtual cloud networks(VCNs) in OCI.
       The complete Oracle Cloud Infrastructure Ansible Modules can be downloaded from
       U(https://github.com/oracle/oci-ansible-modules/releases).
-version_added: "2.8"
+version_added: "2.9"
```

Comment:
Leave the version added as 2.8 -- we'll ignore that CI failure and merge (assuming everything else is OK).

Reply:
@mattclay done